### PR TITLE
BSSwF blog post - visually communicating design

### PR DESCRIPTION
**Pull request by rafmudaf**
This pull request adds a blog post that is part of my 2023 BSSw Fellowship on visually communicating elements of software design. I'd like to start the review process, but there are at least a few things left to do:
- [x] Add captions and numbers to figures
- [x] Refine the layout of the images on the page

Note that some of the content linked in the blog post has not yet been finished, so this pull request should not be merged yet.

### CODE REVIEW

**Comment by bernhold** [comment]
Resolves #2031

**Comment by bernhold** [comment]
Hi @rafmudaf , @gonsie mentioned that you wanted to make some edits to your blog post.  I plan to start work preparing your article for posting next week (the week of 6 May).  It would be most helpful if you could make your changes before then and let me know that it is ready from your perspective.  Thanks

**Comment by rafmudaf** [comment]
Hi @bernhold I've made the changes I discussed with @gonsie.

I think there might be some work to do to improve the readability and layout of the images. Is it straightforward to build the website locally? I watched the [BSSw io Tutorial](https://www.youtube.com/watch?v=FcAwSlcq8eE) video hoping to get some insight into building the docs site, but I think I've included the topics in that video in my process already.

**Comment by bernhold** [comment]
Hi @rafmudaf , unfortunately, there is no capability to build the site locally.  I can work with you on the images.  We do have a preview site that you'll be able to see how they look, but only the BSSw.io editorial team can rebuild it.  So there will be a bit of back and forth.  I'll try to get a preview up for you this weekend and email you when that's ready (we try not to expose the info in public to avoid attracting troublemakers).

**Comment by rafmudaf** [comment]
That sounds good, @bernhold. I'll look out for the email, and I'll try to have the images refined for next week.

**Comment by rafmudaf** [comment]
@bernhold I have a bit more work to do on this post. I saw that you listed the publish date as May 14 in the metadata. Is that my deadline for changes?

**Comment by bernhold** [comment]
@rafmudaf thats a target, but I definitely want to publish it this week.  Please keep me informed.

**Comment by rafmudaf** [comment]
@davidbernholdt I've removed some of the text in that third image, so I think it will ultimately look bigger and a bit more readable.

File: Articles/Blog/2024-05-communicating-design.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+While a strict policy of "docs or it didn't happen" can increase the quality and quantity of documentation, it comes with the added burden for developers, reviewers, and maintainers.
+A more approachable process is to require an extended pull request description that includes a narrative of the changes and an overview of design decisions.
+In any case, including automated diagramming tools in the software infrastructure can reduce the burden
+and empower developers to adopt this practice.
+
+### Summary
+
+Through the BSSw Fellowship, I've had the opportunity to gather and formalize ideas on documentation and communication on software design.
+I presented at the NLIT S3C conference in April 2024 ([slides](https://rafmudaf.github.io/communicating-design/_downloads/67486fd27e6ced8dd8672408a18de874/nlit_s3c.pdf))
+and will present an [HPC Best Practices webinar](https://ideas-productivity.org/resources/series/hpc-best-practices-webinars/) in June 2024.
```
Comment (rafmudaf) [inline Articles/Blog/2024-05-communicating-design.md:102]: @bernhold should this link be replaced with the one for the webinar?
https://ideas-productivity.org/events/hpcbp-084-communicatingdesign

Comment (bernhold) [reply Articles/Blog/2024-05-communicating-design.md:102]: Yeah, please do that.

**Comment by bernhold** [comment]
I've rebuilt the preview for you.

File: Articles/Blog/2024-05-communicating-design.md (hunk 1 above)
Comment (rafmudaf) [reply Articles/Blog/2024-05-communicating-design.md]: Done

**Comment by rafmudaf** [comment]
@bernhold In my view, this is ready for any final reviews.

**Pull request merged**